Servers/WindowServer/Menu.h

```diff
@@ -119,6 +122,8 @@ class Menu final : public Core::Object {
 private:
     virtual void event(Core::Event&) override;
 
+    Gfx::Font* m_title_font { &Gfx::Font::default_font() };
```

Comment:
Let's use a `RefPtr<Gfx::Font>` here so we can't end up with a dangling pointer.